docs/verify-token-access-to-repository.md

```diff
@@ -0,0 +1,65 @@
+## Verify Token Access to Repository
+
+Github PAT token access can be confusing. Here's a quick way to test if the token you're using is authorized to access your repository.
+
+**Remove this snippet after you've verified your token.**
+
+- Make sure you follow the token setup instructions [here](https://github.com/github/issue-metrics/tree/main?tab=readme-ov-file#use-as-a-github-action) first.
+
+- Replace `{owner/repo}` with your own repo information.
+
+- Add this snippet to your workflow.yml.
+
+```
+- name: Check GitHub token permissions
+    run: |
+      curl -H "Authorization: token ${{ secrets.GH_TOKEN }}" https://api.github.com/repos/{owner/repo}
+```
+
+- Go to your repository Actions in Github and run your job.
+- In the job run details, click into the results of `Check Github token permissions`
+- You should see your token details with no errors.
```

Comment:
This is the output in Github actions.
I don't _think_ this opens people up to security issues even if they publish this along with their action, but want a second opinion here @jmeridth @zkoppert 

<img width="948" alt="image" src="https://github.com/user-attachments/assets/88457f82-17f4-4ebe-ae34-7ffc1a6ee641">

Reply:
No security concerns.  GitHub Actions masks the token in the output.  This is a great addition.